sound/soc/codecs/hdac_hda.h

```diff
@@ -6,14 +6,22 @@
 #ifndef __HDAC_HDA_H__
 #define __HDAC_HDA_H__
 
+#define HDAC_ANALOG_DAI_ID		0
+#define HDAC_DIGITAL_DAI_ID		1
+#define HDAC_ALT_ANALOG_DAI_ID		2
+#define HDAC_HDMI_0_DAI_ID		3
+#define HDAC_HDMI_1_DAI_ID		4
+#define HDAC_HDMI_2_DAI_ID		5
```

Comment:
we know we are going to have to deal with 4 converters for TGL so might as well think about it upfront?

Reply:
@plbossart Hmm, true. Once I have the code good to go, I'll make this dynamic as well. Maybe an incremental patch even?